### ISSUE TITLE
Playnite support

### ISSUE THREAD

**Issue by ergoithz** [moraroy/NonSteamLaunchers-On-Steam-Deck#157]
Just discovered Playnite (and installed it inside the nonsteamlaunchers prefix) which greatly simplifies the craziness of thousands of games across dozens of services with actually working third-party integrations (unlike GOG Galaxy).

Font dependencies (not blocking, it just complains in the logs if not available):
- Trebuchet MS
- Consolas

Also, it crashes if ran in desktop app mode, required for configuration, but then gives the option to run in safe mode which works.

Fullscreen mode can be launched directly via dedicated exe providing a controller-friendly UI.

**Comment by moraroy** [comment]
@ergoithz yes I'm aware of those things and have tested extensively for play nite, but as of right now even if you were to set up everything correctly to get play items to run... how would you easily integrate play nite with steam on Linux? The only reason to run play items would be to run the games themselves from playnite or create Steam Shortcuts I've tried to do this with play nite and haven't found a "friendly" way yet for the steam deck if that makes sense. However I did find out that if you install Playnite like how you say, I found that BoilR (Windows) will find all of your games under the Play nite Option. This is a good thing BUT, they aren't correct in the directory and launch options in order to function properly for Steam on linux.

**Comment by moraroy** [comment]
*play items =playnite auto correct
